package.json

```diff
@@ -11,6 +11,9 @@
   "bugs": {
     "url": "https://github.com/firebase/angularfire/issues"
   },
+  "scripts": {
+    "start": "grunt"
```

Comment:
I believe Travis falls back to npm start if you don't have a script specified. When the script was messed up I added this as a failsafe. Happy to remove if you want.